fix(docgen): support @values on classPropHandler

usei95 opened the pull request:
#662

elevatebart commented:
Can you add a unit test in the corresponding file? 
It makes maintenance easier.
Plus, when we want to refactor we can make sure than nothing breaks.

elevatebart commented on packages/vue-docgen-api/src/script-handlers/classPropHandler.ts:
```diff
@@ -53,6 +53,15 @@ export default async function classPropHandler(
+						propDescriptor.values = choices.map((v: string) => v.trim())
+					}
+					delete propDescriptor.tags['values']
+				}
```
Have you tried extracting this into a function? 
I would probably host it in the `propHandler.ts` file.

elevatebart commented:
I created the function and added some tests to propHandler. 
I forgot to create them myself. How could I ask someone else to test?

You can now use the function and write the tests ;-)

Thank you so much

usei95 commented:
> I created the function and added some tests to propHandler.
> I forgot to create them myself. How could I ask someone else to test?
> 
> You can now use the function and write the tests ;-)
> 
> Thank you so much

Thanks, i added test and updated code its now using extractValuesFromTags in propHandler.
